index.js

```diff
@@ -86,45 +88,63 @@ exports.init = function (sbot, config) {
     sbot.publish(content, cb)
   }
 
+  function graph (cb) {
+    onReady(() => {
+      cb(null, layered.getGraph())
+    })
+  }
+
+  function graphStream (opts) {
+    opts = opts || {}
+    const live = opts.live === true // default is false
+    const old = opts.old !== false // default is true
+    if (live) {
+      return pCont((cb) => {
+        onReady(() => {
+          const unsubscribe = layered.onEdge((source, dest, value) => {
+            p.push({ source, dest, value })
+          })
+          const p = Pushable(unsubscribe)
+          if (old) {
+            p.push(layered.getGraph())
+          }
+          cb(null, p)
+        })
+      })
+    } else {
+      return pCont((cb) => {
+        onReady(() => {
+          cb(null, pull.once(layered.getGraph()))
+        })
+      })
```

Comment:
Yeah I don't think this should exist. Just use the sync method right?

Reply:
You think `opts.old` should not be supported at all here?

Reply:
No, this is opts.live = false path
I think if you're not interstate in live, you're just asking fit the current state

So don't use the stream method

Reply:
So that would equate to not having `opts.live`, because it would be always live?

I'm fine with having `opts.live = false` supported because sometimes you may want to concatenate the streams with `pull-cat`, and we have lots of other `opts.live, opts.old` APIs in the SSB stack, that it makes sense to be consistent.

@arj03 tie break?

Reply:
Being a tie breaker feels like a big responsibility. From my point of view I think it makes sense for the API to have similar structure even if some cases are rarely used. It is important that the default options are good, which they are, so I'm in favor of keeping it the way it is.

Reply:
Thanks, I just wanted a third perspective